### ISSUE TITLE
Lua-5.4.4 testrelabelparser.lua:6: assertion failed!

### ISSUE THREAD

**Issue by mingodad** [sqmedeiros/lpeglabel#34]
Trying to test this project with Lua-5.4.4 I'm getting this error:
```
lua-5.4 testrelabelparser.lua 
lua: testrelabelparser.lua:6: assertion failed!
stack traceback:
	[C]: in function 'assert'
	testrelabelparser.lua:6: in function 'testerror'
	testrelabelparser.lua:371: in main chunk
	[C]: in ?
```

**Comment by sqmedeiros** [comment]
The test failed because LPegLabel used to not accept newlines inside a char class. As it now accepts, the test related to an empty char class was not working properly. I did a few modifications and changed the error message related to char classes in order to pass the tests.

**Comment by sqmedeiros** [comment]
Solved the issue in version 1.6.1
